src/test/java/org/broadinstitute/hellbender/IntelDeflaterIntegrationTest.java

```diff
@@ -42,116 +27,7 @@ public void testIntelDeflaterIsAvailable(){
             throw new SkipException("IntelDeflater not available for this architecture");
         }
 
-        Assert.assertTrue(new IntelDeflater().load(), "libIntelDeflater.so was not loaded. " +
+        Assert.assertTrue(new IntelDeflater().load(null), "IntelDeflater shared library was not loaded. " +
                 "This could be due to a configuration error, or your system might not support it.");
     }
-
-    @Test
-    public void deflateInflateWithIntelDeflater() {
```

Comment:
This test still seems valuable to me to have in the GATK (`deflateInflateWithIntelDeflater`), even if it is similar to a test in the GKL. Can you restore it?